core/src/main/java/org/bitcoinj/base/internal/Preconditions.java

```diff
@@ -25,8 +25,7 @@ public class Preconditions {
      * @throws IllegalArgumentException if {@code expression} is false
      */
     public static void checkArgument(boolean expression) {
-        if (!expression)
-            throw new IllegalArgumentException();
+        check(expression, () -> new IllegalArgumentException());
```

Comment:
I think you can use `IllegalArgumentException::new` here.

Reply:
Added.